02-peer-protocol.md

```diff
@@ -408,15 +408,15 @@ Nodes SHOULD send a `closing_signed` message after `shutdown` has
 been received and no HTLCs remain in either commitment transaction.
 
 A sending node MUST set `fee_satoshis` lower than or equal to the
-fee of the final commitment transaction.
+base fee of the final commitment transaction as calculated in [BOLT #3](03-transactions.md#fee-calculation).
 
 The sender SHOULD set the initial `fee_satoshis` according to its
 estimate of cost of inclusion in a block.
 
 The sender MUST set `signature` to the Bitcoin signature of the close
 transaction with the node responsible for paying the bitcoin fee
-paying `fee_satoshis`, without populating any output which is below
-its own `dust_limit_satoshis`. The sender MAY also eliminate its own
+paying `fee_satoshis`, then removing any output which is below
+its own `dust_limit_satoshis`. The sender MAY then also eliminate its own
```

Comment:
@Roasbeef Hmm, that's backwards from what I meant.  Trimming dust doesn't change inputs, so yeah, it effectively ups the fee paid.  This is the same logic we use for commit txs.

But as agreed, will merge, and then the move to BOLT 3, and re-open another PR for clarification if we still need it...